apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss

```diff
@@ -142,7 +142,7 @@ i.close.icon.clear:hover::after {
     bottom: 0;
     cursor: pointer;
     background: $solitude;
-    color: $dim-gray;
+    color: $dark-gray;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine